tools/integration-tests/tester/tests/testutil.go

```diff
@@ -664,13 +722,32 @@ func AwaitTransactionAvailability(peers []*framework.Peer, transactionIDs []stri
 // the the transactions exist beforehand.
 func AwaitTransactionInclusionState(peers []*framework.Peer, transactionIDs map[string]ExpectedInclusionState, maxAwait time.Duration) error {
```

Comment:
since this function is a bit long, can you please either break it up or add some comments?

Reply:
I broke check inclusion state to another function.